frontend/vue/components/Syllabus/SyllabusFormModule.vue

```diff
@@ -0,0 +1,251 @@
+        name="courseContent"
+        color-scheme="light"
+        placeholder="Enter content"
+        @input="updateFormDescription"
+      />
+    </div>
+    <div class="syllabus-form-course__row">
+      <label class="syllabus-form-course__label">
+        Chapters - Click to add qiskit chapters you want show in this section.
+      </label>
+    </div>
+    <div class="syllabus-form-course__row syllabus-form-course__units__container">
+      <div v-for="course in courseList" :key="course.id">
+        <h4 class="syllabus-form-course__units__header">
+          {{ course.title }}
+        </h4>
+        <ColumnFlowGrid class="syllabus-form-course__units__list" :elements="course.sections">
+          <template #default="slotProps">
+            <bx-checkbox
+              :checked="containsUnitID(slotProps.element.uuid)"
+              :label-text="slotProps.element.title"
+              @bx-checkbox-changed="ev => updateFormModuleUnit(slotProps.element.uuid, ev)"
+            />
+          </template>
+        </ColumnFlowGrid>
+      </div>
+    </div>
+    <div class="syllabus-form-course__row syllabus-form-course__row__save">
+      <BasicLink
+        class="syllabus-form-course__link"
+        :class="{'syllabus-form-course__link__disabled': syllabusSaved}"
+        v-bind="saveSyllabusModuleLink"
+        @click="saveModuleAction"
+      >
+        {{ $translate(saveSyllabusModuleLink.label) }}
+      </BasicLink>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue-demi'
+import Close16 from '@carbon/icons-vue/es/close/16'
+import { getCourseList, Course } from '../../../ts/courses'
+import BasicLink from '../common/BasicLink.vue'
+import ColumnFlowGrid from '../common/ColumnFlowGrid.vue'
+import 'carbon-web-components/es/components/button/button.js'
+import 'carbon-web-components/es/components/input/input.js'
+import 'carbon-web-components/es/components/textarea/textarea.js'
+import 'carbon-web-components/es/components/checkbox/checkbox.js'
+import { SyllabusCourse, UnitUUID } from '../../../ts/syllabus'
+import BXTextarea from 'carbon-web-components/es/components/textarea/textarea.js'
+import BXCheckbox from 'carbon-web-components/es/components/checkbox/checkbox.js'
+
+export default defineComponent({
+  name: 'SyllabusFormModule',
+  components: {
+    BasicLink,
+    ColumnFlowGrid,
+    Close16
+  },
+  props: {
+    course: {
+      type: Object,
+      default: () => ({} as SyllabusCourse),
+      required: true
+    },
+    showCloseButton: {
+      type: Boolean,
+      default: false,
+      required: true
+    }
+  },
+  data () {
+    return {
+      courseList: [] as Course[],
+      saveSyllabusModuleLink: {
+        label: 'Save content',
+        url: '#'
+      },
+      syllabusSaved: false
+    }
+  },
+  mounted () {
+    getCourseList().then((courses) => {
+      this.courseList = courses.filter(course => course.type === 'learning-path')
```

Comment:
@korgan00 it looks like we are filtering the courses and only showing those that are part of a learning path? do we not want to include non-learning path content into a syllabus for a specific reason?

@pandasa123 @JRussellHuffman thoughts?

Reply:
I think it should include everything but the `demo` / `example` ones

Reply:
@techtolentino I think this needs to changed to `course.type != 'docs'`?